app/renderer/src/reducers/copiedToClipboard.js

```diff
@@ -0,0 +1,21 @@
+import {
+  UPDATE_QUERY,
+  SELECT_ITEM,
+  SELECT_NEXT_ITEM,
+  SELECT_PREVIOUS_ITEM,
+  COPY_ITEM,
+} from '../actions/types';
+
+export default function (state = false, { type }) {
+  switch (type) {
+    case UPDATE_QUERY:
```

Comment:
Aren't `REQUEST_QUERY` and `RESET_RESULTS` somehow linked to `UPDATE_QUERY`? That's why I did not include them in the first place. Good call for the other ones. Will add 👍